exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "failed";
+  } else if (grade >= 50) {
```

Comment:
else is set because it's the last option, the opposite of grade < 50 would be grade > 50, else return passed would still do the job